bazel/pl_workspace.bzl

```diff
@@ -52,6 +52,11 @@ def _package_manager_setup():
             "libsasl2-2",
             "libssl1.1",
             "libgcc1",
+            "libcrypt1",
+            # TODO(ddelnano): byteman is required until dealing with
+            # netty's libnetty_tcnative.so's random file name is solved for.
+            # See #407 for more details.
+            "libbyteman-java",
```

Comment:
Do you need these ones? Looks like you have them in the thriftmux_base image already.

Reply:
It is needed. This `dpkg_list` is what makes these packages available to install. From my reading of the documentation, this needs to happen in the WORKSPACE file. Maybe it's possible to restructure this into a separate `dpkg_list`, but removing these packages causes the following error when building the thriftmux image

```
vagrant@vagrant:/vagrant$ bazel run src/stirling/source_connectors/socket_tracer/testing/containers/thriftmux:server_image
Starting local Bazel server and connecting to it...
ERROR: Traceback (most recent call last):
        File "/vagrant/src/stirling/source_connectors/socket_tracer/testing/containers/thriftmux/BUILD.bazel", line 143, column 17, in <toplevel>
                packages["libbyteman-java"],
Error: key "libbyteman-java" not found in dictionary
ERROR: Skipping 'src/stirling/source_connectors/socket_tracer/testing/containers/thriftmux:server_image': no such target '//src/stirling/source_connectors/socket_tracer/testing/containers/thriftmux:server_image': target 'server_image' not declared in package 'src/stirling/source_connectors/socket_tracer/testing/containers/thriftmux' defined by /vagrant/src/stirling/source_connectors/socket_tracer/testing/containers/thriftmux/BUILD.bazel
WARNING: Target pattern parsing failed.
ERROR: no such target '//src/stirling/source_connectors/socket_tracer/testing/containers/thriftmux:server_image': target 'server_image' not declared in package 'src/stirling/source_connectors/socket_tracer/testing/containers/thriftmux' defined by /vagrant/src/stirling/source_connectors/socket_tracer/testing/containers/thriftmux/BUILD.bazel
INFO: Elapsed time: 16.452s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (1 packages loaded)
FAILED: Build did NOT complete successfully (1 packages loaded)
```

Reply:
Makes sense.